### ISSUE TITLE
Schema: Ergänzungen zu den Metadaten

### ISSUE THREAD

**Issue by susannehaaf** [deutschestextarchiv/dtabf#6]
Ziel: Sichtbarkeit der Daten im VLO erhöhen
Inhalt (Vorschlag):
* Modality: `//profileDesc/textDesc/channel[@mode='w'][text() = 'printed text']`
* Keywords?: `//profileDesc/keywords`
* Short description of resource: `//profileDesc/abstract`

**Comment by xlhrld** [comment]
Die vorgeschlagenen Felder finde ich aus TEI-Sicht sinnvoll. Um das vom VLO ausgewertet zu bekommen , muss dann aber noch das CMDI-Profil entsprechend erweitert und annotiert werden. Müsstest du dann wohl machen, @susannehaaf.

+1 von mir

**Comment by xlhrld** [comment]
Nur zur Sicherheit: _Short description of resource_ im VLO soll heißen: kurze inhaltliche Zusammenfassung? Dann passt `abstract`. Wenn es eher eine um eine technische Beschreibung geht (»Dieses Buch aus dem 17. Jahrhundert wurde nach den DTA-Richtlinien …«), dann besser `editionStmt/p` oder `notesStmt/note`. Ich tippe aber auf ersteres, da die technischen Details ja bereits recht ausführlich an anderer Stelle kodiert sind.

**Comment by susannehaaf** [comment]
@xlhrld 
Ja, CMDI-Anpassung hängt da mit dran. Einen "Prototypen" gibt es schon, den ich aber erst finalisieren würde, wenn wir auf DTABf-Seite die Änderungen festgelegt haben. 

Bzgl. Abstract ist eher die "technische" Beschreibung gemeint. Ich kann mit beiden Alternativvorschlägen leben, mit einer leichten Präferenz für `editionStmt/p`.

**Comment by xlhrld** [comment]
`editionStmt/p` sollte es werden, das sehe ich auch so. Schließlich geht es ja um die Eigenschaften genau der elektronischen Ausgabe im DTA.

**Comment by textloop** [comment]
editionStmt/p wäre auch meine Wahl.

**Comment by susannehaaf** [comment]
Ich stelle gerade fest, dass `editionStmt/p` leider nicht mehr geht, weil wir schon `editionStmt/edition` haben. Somit wäre `abstract` wieder auf dem Tisch. Vielleicht ließe es sich rechtfertigen, wenn es etwas allgemeiner formuliert wäre: "Historical German text source (16th—19th c.); part of the Deutsches Textarchiv (German Text Archive) collection." ?
